version/6_14_3.sh

```diff
@@ -1,8 +1,7 @@
 #!/bin/bash -e
 
 NODE_VERSION=6.14.3
-NPM_VERSION=5.6.0
 echo "=============== Installing Node $NODE_VERSION ============="
 . /root/.nvm/nvm.sh && nvm install $NODE_VERSION
-nvm use $NODE_VERSION && npm install npm@$NPM_VERSION -g
+nvm use $NODE_VERSION &&  npm install npm@6.3.0 -g
```

Comment:
we should export this as a variable like `NODE_VERSION`

Reply:
ok